app/assets/javascripts/users/promote.js.coffee

```diff
@@ -0,0 +1,6 @@
+$(document).ready ->
+  $ ->
```

Comment:
I'm not sure this line (and its accompanying `return`) is necessary. Can we compare this with coffee script in other repos?